images/nginx/build.sh

```diff
@@ -85,20 +85,41 @@ clean-install \
   libcurl4-openssl-dev \
   procps \
   git g++ pkgconf flex bison doxygen libyajl-dev liblmdb-dev libtool dh-autoreconf libxml2 libpcre++-dev libxml2-dev \
+  lua-cjson \
   python \
   luarocks \
   || exit 1
 
-ln -s /usr/lib/x86_64-linux-gnu/liblua5.1.so /usr/lib/liblua.so
+if [[ ${ARCH} == "x86_64" ]]; then
+  ln -s /usr/lib/x86_64-linux-gnu/liblua5.1.so /usr/lib/liblua.so
+  ln -s /usr/lib/x86_64-linux-gnu /usr/lib/lua-platform-path     
+fi
 
-mkdir -p /etc/nginx
+if [[ ${ARCH} == "armv7l" ]]; then
```

Comment:
We are not building armv71 https://github.com/toolboc/ingress-nginx/blob/ede1094f6d212b5f58d409eeed4d3c9b5747afdb/images/nginx/Makefile#L21

Reply:
@aledbf 

[build.sh](https://github.com/toolboc/ingress-nginx/blob/ede1094f6d212b5f58d409eeed4d3c9b5747afdb/images/nginx/build.sh#L42) is obtaining the platform using `ARCH=$(uname -m)`  which returns 'armv71' in nginx-arm

![image](https://user-images.githubusercontent.com/2018336/40934821-a75d93b0-67fb-11e8-9616-9c1897acc6d4.png)